<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZQD</class>
 <widget class="QMainWindow" name="ZQD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ZQD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>750</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1600</width>
      <height>750</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_File_N"/>
   <addaction name="actionOpen_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionShow_Dock_D"/>
   </widget>
   <addaction name="menuFile_F"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>41</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>显示效果</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>输入填充字符</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>选择字体</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>输入打印字符</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>FONT</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/image/images/fileopen.png</normaloff>:/image/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew_File_N">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/image/images/filenew.png</normaloff>:/image/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New File(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionShow_Dock_D">
   <property name="text">
    <string>Show Dock(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myimages.qrc"/>
 </resources>
 <connections/>
</ui>
